source/administration-guide/onboard/migrate-from-slack.rst

```diff
@@ -4,29 +4,118 @@ Migrate from Slack
+1. `**Preparations** <#1.-preparations>`_:
+
+   - Answer key scoping questions.
+   - Gather environment and export details.
+   - Validate Mattermost server capacity and configuration.
+   - Back up your Mattermost environment before importing.
+
+2. `**Export Slack data** <#2.-export-slack-data>`_:
+
+   - Generate an export from Slack in the user interface or by contacting Slack directly.
+   - Download file attachments and email addresses using the ``slack_advanced_exporter``
+
+3. `**Transform the export for Mattermost** <#3.-transform-the-export-for-mattermost>`_:
+
+   - Use the ``mmetl`` tool to parse and transform Slack exports.
+
+4. `**Import into Mattermost** <#4.-import-data-into-mattermost>`__:
+
+   - Upload and process transformed archives with ``mmctl``.
+
+5. **Validate and test**:
+
+   - Confirm channel and user data imported correctly.
+   - Run database queries to fix any unread states.
+
+6. **Go live**:
+
+   - Communicate the cutover plan to users.
+
+
+Migration timeline
+~~~~~~~~~~~~~~~~~~
+
+These instructions outline a *best effort* migration path designed to preserve the majority of your messages, files, and workspace structure. Mattermost provides tools and guidance to help streamline the process, but manual adjustments during the data transformation and import steps are often required. Successful migration depends on careful planning and dedicating sufficient time, technical resources, and technical skills to the effort.
+
+Depending on the size and complexity of your Slack environment, a full migration can take anywhere from several days to multiple weeks of dedicated effort. Larger organizations with multiple workspaces, extensive message history, and many files should expect the process to require significant iteration and testing before completion. It’s important to plan for this timeline in advance by allocating the necessary resources, scheduling time for trial imports in a development environment, and coordinating across teams. Building in extra time for adjustments during the transformation and import steps will help ensure a smoother transition and reduce disruption to your users.
+
+Scoping the migration appropriately during the preparation step can significantly reduce processing time and allow for faster iteration. Before beginning, carefully consider what data is essential to bring over to Mattermost. Many organizations find that not every channel or file needs to be migrated, and focusing only on what is truly needed can save substantial processing time and manual effort. By setting clear boundaries early, you’ll minimize the amount of data that requires manual intervention and testing, which in turn shortens the migration timeline and helps avoid unnecessary complexity.
+
+
 .. note::
+  Consider `talking to a Mattermost expert <https://mattermost.com/contact-sales/>`_ if your organization needs support migrating from Slack to Mattermost.
+
+
+Migrations Steps
+----------------
+
+
+1. Preparations
```

Comment:
There's a mix of Sentence case and Title Case throughout this docs page. Please choose one and apply it consistently